addons/website_form/views/snippets/s_website_form.xml

```diff
@@ -18,6 +18,11 @@
 </template>
 
 <template id="s_website_form_options" inherit_id="website.snippet_options">
+    <!-- Exclude backgound options for inline forms -->
+    <xpath expr="//t[@t-set='both_bg_color_image_exclude']" position="after">
+        <t t-set="both_bg_color_image_exclude" t-value="both_bg_color_image_exclude + ', .row .s_website_form, .col .s_website_form'"/>
+    </xpath>
```

Comment:
Why only for website_form? Why not the same selector as the other, to exclude all snippets dropped into columns?

Reply:
I was thinking that maybe we do not want to remove the background color and image options for every snippet in a column, maybe we want to keep the option for changing the background of a banner in a column for example. But I don't know if it makes sense ?

Reply:
A banner in a column? That's a thing? No, every "main" snippet (= that can be dropped in the page) but that can also be dropped in a column should not have these background options :+1:

Reply:
It is possible on the contact us page, but I guess it's not supposed to happen otherwise, I will add it to the `both_bg_color_image_exclude` selector